src/content/history/index.md

```diff
@@ -9,13 +9,17 @@ sidebarDepth: 1
 
 A timeline of all the major milestones, forks, and updates to the Ethereum blockchain.
 
-<ExpandableCard title="What are forks?" contentPreview="Changes to the rules of the Ethereum protocol which often include planned technical upgrades.">
-
-Forks are when major technical upgrades or changes need to be made to the network – they typically stem from [Ethereum Improvement Proposals (EIPs)](/eips/) and change the "rules" of the protocol.
+<ExpandableCard>
+  <ExpandableCardPreview title="What are forks?">
+    Changes to the rules of the Ethereum protocol which often include planned technical upgrades.
+  </ExpandableCardPreview>
+  <ExpandableCardContent>
+  Forks are when major technical upgrades or changes need to be made to the network – they typically stem from [Ethereum Improvement Proposals (EIPs)](/eips/) and change the "rules" of the protocol.
```

Comment:
This works as expected in Crowdin 👍 nice workaround @pettinarip.

![Screenshot 2023-05-17 at 16 22 02](https://github.com/ethereum/ethereum-org-website/assets/62268199/8f082db0-5365-45f5-abf2-02652b755d63)

One question, might we want to do the same with the title? This makes the context clearer in Crowdin, but mostly because it feels weird mixing the two methods of declaring the card's content.

```suggestion
<ExpandableCard>
<ExpandableCardTitle>
    What are forks?
  </ExpandableCardTitle>
  <ExpandableCardPreview>
    Changes to the rules of the Ethereum protocol which often include planned technical upgrades.
  </ExpandableCardPreview>
  <ExpandableCardContent>
  Forks are when major technical upgrades or changes need to be made to the network – they typically stem from [Ethereum Improvement Proposals (EIPs)](/eips/) and change the "rules" of the protocol.
```

Reply:
Oh right, we can do that, yea. I agree that it will be cleaner, good point.

Just for curiosity, does the `title` attr content works with Crowdin? because I don't see it in your example. The text `"What are forks?"` does not appear on the screen that you shared 🤔

Reply:
I think what @minimalsm is proposing is create another component `ExpandableCardTitle` and pass the text in there instead of passing a prop to `ExpandableCard` or `ExpandableCardPreview`. Not sure if that clears that up, or if there was confusion before 😅